frontend/vite.config.ts

```diff
@@ -1,9 +1,48 @@
+import { readFileSync } from 'node:fs'
 import { resolve } from 'node:path'
 import { execSync } from 'child_process'
 import { defineConfig, loadEnv } from 'vite'
 import solidPlugin from 'vite-plugin-solid'
 import unocssPlugin from 'unocss/vite'
 
+function eruda(debug = undefined) {
+  const erudaSrc = readFileSync("./node_modules/eruda/eruda.js", "utf-8");
+  return {
+    name: "vite-plugin-eruda",
+    transformIndexHtml(html) {
+      const tags = [
+        {
+          tag: "script",
+          children: erudaSrc,
+          injectTo: "head",
+        },
+        {
+          tag: "script",
+          children: "eruda.init();",
+          injectTo: "head",
+        },
+      ];
+      if (debug === true) {
+        return {
+          html,
+          tags,
+        };
+      } else if (debug === false) {
+        return html;
+      }
+      // @ts-ignore
+      if (process.env.NODE_ENV !== "production") {
```

Comment:
can't you merge this with the previous ifs?

Reply:
no, because if `debug` is set it takes priority over the value of `NODE_ENV`, but anyway we don't need that debug parameter as we want to inject Eruda dynamically not forcing it in the `vite.config.ts` file so I will just remove the debug parameter and always check `NODE_ENV`